packages/eslint-plugin-office-addins/src/utils.ts

```diff
@@ -173,3 +178,89 @@ export function isLoadFunction(node: TSESTree.MemberExpression): boolean {
     node.property.name === "load"
   );
 }
+
+export function isContextSyncIdentifier(node: TSESTree.Identifier): boolean {
+  return (
+    node.name === "context" &&
+    node.parent?.type === TSESTree.AST_NODE_TYPES.MemberExpression &&
+    node.parent?.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression &&
+    node.parent?.property.type === TSESTree.AST_NODE_TYPES.Identifier &&
+    node.parent?.property.name === "sync"
+  );
+}
+
+export type OfficeApiReference = {
+  operation: "Read" | "Load" | "Write" | "Sync";
+  reference: Reference;
+};
+
+export function isLoadReference(node: TSESTree.Identifier) {
+  return (
+    node.parent &&
+    node.parent.type === TSESTree.AST_NODE_TYPES.MemberExpression &&
+    isLoadFunction(node.parent)
+  );
+}
+
+let proxyVariables: Set<Variable>;
+let apiReferences: OfficeApiReference[];
+export function findReferences(scope: Scope): OfficeApiReference[] {
```

Comment:
Maye this should be named "findOfficeApiReferences" since it's being made a util function (no contest to interpret the name with).